### PR TITLE
kristal-px-157-update-features-section-docs

### PR DESCRIPTION
**Pull request by KGmajor**
Updating `features` section to be less instructional and more similar in language to our marketing materials. All instructions on use and configurations will move to guides and advanced configuration sections.

### CODE REVIEW

File: docs/features/alerts.md [hunk 1]
```diff
@@ -1,4 +1,17 @@
...
+---
+# Alerting
+
+With Datafold's alerts you'll be able to quickly monitor the data flowing through your pipelines.&#x20;
+
+![](https://i.imgur.com/uAef9ua.gif)
+
+Turn any SQL query into an alert that emails or messages your team. Get faster incident response times with code-free alerting that are customizable to your data.
+
+Datafold’s model learns how your data works to adapt to the seasonality and trends in your data to create dynamic thresholds with anomoly detection. Adust the sensitivity or add custom anomaly detection to optimize your signal-to-noise ratio.
```
Comment (dlawin) [inline docs/features/alerts.md:13]: anomoly -> anomaly

Comment (dlawin) [reply docs/features/alerts.md:13]: adust -> adjust

File: docs/features/catalog-lineage.md [hunk 2]
```diff
@@ -1,4 +1,16 @@
...
\ No newline at end of file
+---
+## Catalog
+![](../../static/img/catalog_landing.png)
+With the Datafold Catalog you can quickly see row counts, nulls and column distributions. We'll gather up this information and display it for every table and data source that you add. 
+
+For more detail, you can sync metadata from dbt and your data warehouse to create a single source of truth. You can implement custom tags to easily categorize and filter the data sets. 
+
+## Column-level Lineage
+After connecting your data warehouse Datafold analyzes every SQL statement produces the graph of dependencies to easily visualize your data. See how data is produced and consumed - even correlated subqueries, CASE WHEN statements, and other complex queries are covered.
```
Comment (dlawin) [inline docs/features/catalog-lineage.md:12]:
```suggestion
After connecting your data warehouse, Datafold analyzes every SQL statement and produces the graph of dependencies to easily visualize your data. See how data is produced and consumed - even correlated subqueries, CASE WHEN statements, and other complex queries are covered.
```

**Review by dlawin — APPROVED** [review]

**Comment by KGmajor** [comment]
Thanks @dlawin! I also turned on my spellcheck extension 😂

**Comment by williebsweet** [comment]
I had a question about hosting the gif in static assests. See above

**Comment by williebsweet** [comment]
I also think we should use oxford commas in our docs.

**Pull request merged**